app/views/arcade_machines/show.html.erb

```diff
@@ -1,40 +1,84 @@
-<div class="arcade-machine">
-  <% if current_user&.admin? || current_user.owns?(@arcade_machine) %>
-    <%= link_to "Edit", edit_arcade_machine_path(@arcade_machine), class: "pull-right" %>
-  <% end %>
-
-  <%= page_heading :h1, @arcade_machine.title %>
-  <div class="row well">
-    <p><%= @arcade_machine.location %></p>
-    <%= simple_format(@arcade_machine.description) %>
-  </div>
+<!--
+  GAME DETAIL VIEW
+-->
 
-  <% if current_user&.admin? || current_user.owns?(@arcade_machine) %>
-    <div class="row">
-      <h4>API Key</h4>
-      <pre><%= @arcade_machine.api_keys.first.token %></pre>
-    </div>
-  <% end %>
+<%= render "shared/detail_header", image: @arcade_machine.cover_image.url, title: @arcade_machine.title, description: @arcade_machine.description %>
 
-  <div class="row">
-     <%= render "images/list", imageable: @arcade_machine %>
-  </div>
+<div class="arcade-machine detail">
 
   <div class="row">
-    <h4>Those responsible</h4>
-    <%= render "shared/user_cards", users: @arcade_machine.users %>
-  </div>
 
-  <h3>Playlists</h3>
 
-  <table class="table table-condensed hover">
-    <% @arcade_machine.playlists.each do |playlist| %>
-      <% subscription = Subscription.find_by(playlist: playlist, arcade_machine: @arcade_machine) %>
-      <tr>
-        <td><%= link_to playlist.title, playlist %></td>
-        <td><%= link_to "unsubscribe", subscription_path(subscription), method: :delete %></td>
-      </tr>
-    <% end %>
-  </table>
 
-</div>
\ No newline at end of file
+    <!-- Metadata -->
+
+    <div class="col-md-6">
+
+      <!-- Game Description -->
+      <div class="detail-description">
+        <%= simple_format(@arcade_machine.description) %>
+
+        <!-- Edit Buttons -->
+        <div class="detail-edit-buttons">
+          <%= render "edit_buttons", arcade_machine: @arcade_machine %>
+        </div>
+      </div>
+
+      <hr />
+
+      <h4>Location</h4>
+      <p><%= @arcade_machine.location %></p>
+
+      <h4>Images</h4>
+      <%= render "images/list", imageable: @arcade_machine %>
+
+      <h4>
+        <%= @arcade_machine.users.size > 1 ? "Owners" : "Owner" %>
+      </h4>
+
+      <div class="clearfix">
+        <% @arcade_machine.users.each do |user| %>
+          <%= render "shared/item_cards_square",
+            itemName: user.name,
+            itemInfo: "",
+            itemLink: user_url(user),
+            itemImageURL: user.gravatar_url
```

Comment:
We'll leave it for now, but for future reference it's more idiomatic in Ruby to use_snake_case rather than camelCase.